.github/workflows/build.yml

```diff
@@ -25,6 +25,11 @@ jobs:
         with:
           dotnet-version: '6.0.x'
 
+      - name: Install .NET SDK 7.0.x
```

Comment:
Why is this required?

Reply:
It installs latest .NET 7 SDK as global json will install .NET 8.0.x SDK,

But it broke many things, so will revert and take it with .NET 8 support PR.